app/src/main/java/com/iti/android/team1/ebuy/ui/productsScreen/ProductsRecyclerAdapter.kt

```diff
@@ -52,7 +53,7 @@ class ProductsRecyclerAdapter(
                 }
 
                 override fun unLiked(likeButton: LikeButton) {
-                    onUnLike(products.products!![bindingAdapterPosition])
+                    onUnLike(products.products!![bindingAdapterPosition].productID!!)
```

Comment:
It is not recommended to use a not-null assertion operator

Reply:
done